build-system/tasks/bundle-size/index.js

```diff
@@ -170,7 +166,7 @@ async function skipBundleSize() {
  */
 async function reportBundleSize() {
   if (isPullRequestBuild()) {
-    const baseSha = gitCiMasterBaseline();
+    const baseSha = gitSha('origin/master');
```

Comment:
Super²nit: does it have to be `origin/master`? If we just set it to `master` would it work just the same?
Since this function is only supposed to be called during CI, basically the only difference is for us when we try to debug this locally - I can see myself wasting half a day not understanding why this doesn't work only to realize it's hardcoded for `origin` instead of `upstream`

Reply:
Sure, I agree with this. I'm going to wait for the test runs to complete before changing.